packages/blade/src/components/Table/Table.web.tsx

```diff
@@ -87,21 +110,42 @@ const getTableHeaderCellCount = (children: (data: []) => React.ReactElement): nu
   return 0;
 };
 
-const StyledReactTable = styled(ReactTable)<{ $styledProps?: { height?: BoxProps['height'] } }>(
-  ({ $styledProps }) => {
-    const { theme } = useTheme();
-    const styledPropsCSSObject = getBaseBoxStyles({
-      theme,
-      height: $styledProps?.height,
-    });
+const StyledReactTable = styled(ReactTable)<{
+  $styledProps?: {
+    height?: BoxProps['height'];
+    width?: BoxProps['width'];
+    isVirtualized?: boolean;
+    isSelectable?: boolean;
+    showStripedRows?: boolean;
+  };
+}>(({ $styledProps }) => {
+  const { theme } = useTheme();
+  const styledPropsCSSObject = getBaseBoxStyles({
+    theme,
+    height: $styledProps?.height,
+    ...($styledProps?.isVirtualized && {
+      width: $styledProps?.width,
+    }),
+  });
+  const $isSelectable = $styledProps?.isSelectable;
+  const $showStripedRows = $styledProps?.showStripedRows;
 
-    return {
-      '&&&': {
-        ...styledPropsCSSObject,
-      },
-    };
-  },
-);
+  return {
+    '&&&': {
+      ...styledPropsCSSObject,
+    },
+    ...($styledProps?.isVirtualized && {
+      ...getTableBodyStyles({
```

Comment:
You don't have to destructure here right? I'm guessing it can just be `....(isVirtualized ? getTableBodyStyles() : {})`

Reply:
So in virtualized version, we add tableBodyStyles on table component?

Is there any harm if we always use tableBodyStyles here? even in non-virtualized table?

Reply:
nope, since structure is different i don't see any harm . but still we should not add any other styles which are not being used.